src/genie/libs/parser/iosxe/tests/ShowCtsApSgtInfo/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,13 @@
+expected_output = {
+    "number_of_sgts_referred_by_the_ap": 2,
+    "sgts": {
+        "UNKNOWN(0)": {
+            "policy_pushed_to_ap": "no",
```

Comment:
please take the value as is.
```suggestion
            "policy_pushed_to_ap": "NO",
```

Reply:
Updated